internal/doc/butane.yaml

```diff
@@ -0,0 +1,439 @@
+                    # no inline directive in fcos 1.0.0
+                    - regex: " Mutually exclusive with `inline`."
+                      replacement: ""
+                      if:
+                        - variant: fcos
+                          max: 1.0.0
+            - name: replace
+              children:
+                - name: source
+                  # first merge with the Ignition root needs to add the
+                  # field; the merge with the component happens afterward
+                  after: $
+                  transforms:
+                    # no inline directive in fcos 1.0.0
+                    - regex: " Mutually exclusive with `inline`."
+                      replacement: ""
+                      if:
+                        - variant: fcos
+                          max: 1.0.0
+        - name: security
+          children:
+            - name: tls
+              children:
+                - name: certificate_authorities
+                  transforms:
+                    - regex: "unique `source`"
+                      replacement: "$0, `inline`, or `local`"
+                    # and then undo it for fcos 1.0.0
+                    - regex: ", `inline`, or `local`"
+                      replacement: ""
+                      if:
+                        - variant: fcos
+                          max: 1.0.0
+                  children:
+                    - name: source
+                      transforms:
+                        # no inline directive in fcos 1.0.0
+                        - regex: " Mutually exclusive with `inline`."
+                          replacement: ""
+                          if:
+                            - variant: fcos
+                              max: 1.0.0
+                    - name: inline
+                      # first merge with the Ignition root needs to add the
+                      # field; the merge with the component happens afterward
+                      after: $
+                      transforms:
+                        - regex: "%TYPE%"
+                          replacement: "certificate bundle (in PEM format). The bundle can contain multiple concatenated certificates"
+                    - name: local
+                      # first merge with the Ignition root needs to add the
+                      # field; the merge with the component happens afterward
+                      after: $
+                      transforms:
+                        - regex: '%TYPE%(, relative to the[^.]+)\.'
+                          replacement: "certificate bundle (in PEM format)$1. The bundle can contain multiple concatenated certificates."
+        - name: proxy
+          transforms:
+            # Snake-case cross-references between children
+            - regex: "`(https|no)Proxy`"
+              replacement: "`${1}_proxy`"
+              descendants: true
+    - name: storage
+      children:
+        - name: disks
+          unless:
```

Comment:
There are 24 of these directives and they clutter things up.  I'd like to look at deriving this info automatically, but that'll require some refactoring and I'll save it for a followup.